src/holo-test

```diff
@@ -39,9 +39,9 @@ run_testcase() {
     # setup chroot for holo run
     rm -rf -- target/
     cp -R source/ target/
+    mkdir -p target/run
+    mkdir -p target/tmp
     mkdir -p target/usr/share/holo/files
-    mkdir -p target/usr/share/holo/run-scripts
-    mkdir -p target/usr/share/holo/users-groups
```

Comment:
Oh! That's a good solution!  I think `plugin [^=]\+` fits the syntax better, and it would need to also include `holorc.d`.  I'll update the patch now.